public/keymaps/d/dp3000_default.json

```diff
@@ -0,0 +1,57 @@
+{
+  "keyboard": "dp3000",
+  "keymap": "default",
+  "commit": "80d679fac3406d9c3d6f3b10e133161c3f938697",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_MUTE",
+      "MO(1)",
+      "KC_1",
+      "KC_2",
+      "KC_3",
+      "KC_4",
+      "KC_5",
+      "KC_6",
+      "KC_7",
+      "KC_8"
+    ],
+    [
+      "QK_BOOT",
+      "RGB_VAI",
+      "RGB_RMOD",
+      "RGB_MOD",
+      "RGB_HUI",
+      "RGB_SAI",
+      "TO(0)",
+      "TO(2)",
+      "TO(3)",
+      "RGB_TOG"
+    ],
+    [
+      "KC_NO",
+      "MO(1)",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO"
+    ],
+    [
+      "KC_NO",
+      "MO(1)",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO",
+      "KC_NO"
+    ]
```

Comment:
This works, but it doesn't match the key assignment given when compiling from the `qmk_firmware` repository, which means users will experience different behaviour depending on how they compile their keymaps.

https://github.com/qmk/qmk_firmware/blob/632de46b006a03ff04095a00ab21179d9510e39a/keyboards/dp3000/keymaps/default/keymap.c#L35-L49

Reply:
im sorry for the inconsistent key assignment, i have resolved the issue by making the necessary changes to match the original `keymap.c` from the qmk_repository